api/KeyboardEvent.json

```diff
@@ -951,10 +951,10 @@
                 "version_added": "35"
               },
               "safari": {
-                "version_added": null
+                "version_added": false
```

Comment:
Confirmed by "FIXME: The specification also has Fn, FnLock, Hyper, NumLock, Super, ScrollLock, Symbol, SymbolLock." comment.